content/blog/experiences-with-ci-cd.pt-br.md

```diff
@@ -0,0 +1,476 @@
+páginas enquanto testava as capacidades de template do Hugo --- a versão
+instalada na minha máquina era diferente da instalada no servidor.
+
+A falta de um ambiente facilmento reproduzido para testes (e obviamente os
+testes em si) significava que todo deployment feito arriscava quebrar a
+"produção".
+
+**Razão #3:** Mudanças não passam por nenhum processo de revisão.
+
+Para várias pessoas, revisão de código não está necessariamente dentro do escopo
+de integração contínua, mas na minha humilde opinião, é central para esta. Se CI
+tem o intuito de controlar ou garantir um grau de qualidade antes de mudanças
+entrarem em branches estáveis, revisões estão perfeitamente alinhadas com ela, e
+ficam no caminho da integração.
+
+Eu disse antes que _acredito que CI pode ser atingida mesmo quando há somente um
+contribuidor,_ então por que levanto esse ponto? A razão é que eu não revisava
+minhas próprias mudanças. Eu as testava localmente, claro, mas não as revisava
+com imparcialidade depois do desenvolvimento. A falta de qualquer processo de
+revisão, ao meu ver, desclassifica esta esteira de receber o apelido de
+integração contínua.
+
+[atlassian-ci]: https://www.atlassian.com/continuous-delivery/continuous-integration
+
+### Por que houve Deployment Contínuo (talvez)
+
+Se você espremer bem os olhos, ou for permissivo no acoplamento entre CI e CD, o
+setup com Git & Make tinha, sim, deployment contínuo. Mais uma vez, começamos
+com uma definição. Dessa vez, ela vem do [TechTarget:][techtarget-cd]
+
+> Deployment Contínuo (CD) é uma estratégia de lançamento de software onde toda
+> commit que passa pela fase de teste automatizado é automaticamente lançada no
+> ambiente de produção, fazendo mudanças visíveis aos seus usuários.
+
+Commits que chegavam à `master`, usada para produção no nosso caso, chegariam
+automaticamente ao ambiente de produção --- a versão do site servida pelo
+servidor. Nesse sentido, isso foi um exemplo de deployment contínuo, mas _se não
+há CI, pode haver CD?_ É necessária a presença dessa "fase de teste automático"
+para que o deployment seja contínuo?
+
+Eu diria que não. CI e CD são ambos premissas fundamentais da cultura DevOps,
+mas a implantação dos dois costuma ser feita separadamente. Fossem eles
+genuinamente inseparáveis, provavelmente teríamos uma expressão só a designar a
+"unidade" composta por ambos.
+
+[techtarget-cd]: https://www.techtarget.com/searchitoperations/definition/continuous-deployment
+
+## O caminho para CI/CD propriamente dito
+
+Meus olhos se abriram para CI/CD quando comecei a trabalhar. Até então, mal
+sabia o que significavam, se é que havia realmente ouvido falar dos dois. A mim,
+CI era a segunda parte de [Travis CI][travis-ci], uma ferramenta que rodava
+builds no repositório do XMonad de forma automática, e CD era um formato de
+mídia ótica.
+
+Começo a trabalhar e quase que de imediato, me mostram o [Jenkins][jenkins]. Me
```

Comment:
"Foi apresentado a mim"

Reply:
Since I used the verb "apresentar" right afterwards, I went with "me deparo com" in ead6a2d to avoid repetition. What do you think?